deploy/docs/Best_Practices.md

```diff
@@ -246,16 +246,17 @@ you will need to ensure the logs get mounted to the host so fluent-bit can be
 configured to capture from the host.
 
 Example:
-In `values.yaml` in the `fluent-bit.rawConfig` section, you have to add
-a new input specifying the file path, eg.
+In `values.yaml` in the `fluent-bit.config.input` section, you have to add
+a new `INPUT` specifying the file path (retaining the remaining part of `input`
+config), e.g.:
 
-```bash
+```
```

Comment:
```suggestion
```yaml
fluent-bit:
  config:
    input: |-
      # Here should be copied original fluent-bit.config.input
```

and indentation for the section